src/app/components1/Types/Product.ts

```diff
@@ -0,0 +1,10 @@
+interface Product {
```

Comment:
**suggestion:** Follow the [Angular Style Guide](https://angular.io/guide/styleguide#general-naming-guidelines) for naming files, classes and variables.

In this case the file should be named `product.ts`